oracledb-ballerina/types.bal

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java;
+import ballerina/sql;
+
+# Structure of  NTERVAL YEAR TO MONTH.
+# + year - Number of years
+# + month - Number of months
+type IntervalYearToMonthRecord record {|
+    int|string year;
+    int|string month;
+|};
+
+# Structure of NTERVAL DAY TO SECOND.
+# + day - Number of days
+# + hour - Number of hours
+# + minute - Number of minutes
+# + second - Number of seconds
+type IntervalDayToSecondRecord record {|
+    int|string day;
+    int|string hour;
+    int|string minute;
+    float|string second;
+|};
```

Comment:
is it possible to use `time:Date` and `time:TimeOfDate` type for these

Reply:
Will try out

Reply:
In `time:Date` all year, month and date are required. But only date is there in `IntervalDayToSecondRecord`. But time:TimeOfDate can be used to keep the other three params (hours, minutes, and seconds)